src/openai/lib/streaming/chat/_completions.py

```diff
@@ -434,6 +434,7 @@ def _accumulate_chunk(self, chunk: ChatCompletionChunk) -> ParsedChatCompletionS
                 choice_snapshot.message.content
                 and not choice_snapshot.message.refusal
                 and is_given(self._rich_response_format)
+                and choice_snapshot.message.content.strip() # Add filter to ensure no parsing errors
```

Comment:
nit: we only care about leading whitespace so this is more performant
```suggestion
                and choice_snapshot.message.content.lstrip() # Add filter to ensure no parsing errors
```